website/content/docs/secrets/pki/cieps.mdx

```diff
@@ -0,0 +1,313 @@
+   Vault; a compatible response format is expected.
+
+ - `request_uuid` `(string)` - A random UUID which serves to identify this
+   request. This value must be sent in the response.
+
+ - `synchronous` `(bool: true)` - A boolean indicating whether the request is
+   synchronous or not. Presently set to true; no asynchronous response is
+   understood.
+
+ - `user_request_key_values` `(map[string]interface{})` - The unvalidated
+   request parameters sent by the user. It is up to the CIEPS service to
+   validate these prior to using them. The following fields may be present,
+   including any other fields submitted by the user:
+
+   - `csr` `(string)` - A PEM format CSR submitted either by the client (
+     in the case of `/sign` or ACME requests) or on the client's behalf
+     (in the case of `/issue` requests, where key material is generated
+     by Vault).
+
+ - `identity_request_key_values` `(map[string]interface{})` - Values related
+   to the user's identity. When the request type is ACME, this value is not
+   populated. These are:
+
+   - `entity_id` `(string)` - The entity identifier from the request after
+     authentication.
+
+   - `entity` `(map[string]interface{})` - The entire resolved `logical.Entity`
+     of the user after authentication; subject to change by the
+     `entity_jmsepath` parameter in the configuration.
+
+   - `groups` `([]map[string]interface{})` - The set of resolved
+     `logical.Groups` of the user after authentication; subject to change by
+     the `group_jmsepath` parameter in the configuration.
+
+   ~> **Note**: in the event that the direct token backend or a root token is
+      used, entity information may not exist. In either case,
+      `identity_request_key_values` will be omitted.
+
+ - `acme_request_key_values` `(map[string]interface{})` - Values related to
+   ACME authorizations challenges attached to the finished order. Only present
+   when the request type is ACME. These are:
+
+   - `authorizations` `(map[string]interface{})` - Authorizations and
+     challenges solved by the client to move this order to the finalization
+     state.
+
+   - `account` `(map[string]interface{})` - Information related to the ACME
+     account issuing the request. These are:
+
+     - `id` `(string)` - The UUID of the ACME account.
+
+     - `directory` `(string)` - The path to the ACME directory requested by
+       this account.
+
+     - `contact` `([]string)` - Unverified contact information submitted by
+       the requesting ACME account on creation.
+
+     - `created_date` `(string: RFC 3999 format)` - Timestamp when the account
+       was created.
+
+     - `eab` `(map[string]interface{}, optional)` - When present, the details
+       of the EAB used to authorize this account via Vault authentication. If
+       not present, this ACME account was created without EAB bindings.
+
+     - `key_id` `(string)` - Identifier of the EAB binding used by this
```

Comment:
shouldn't `key_id`, `key_type`, `created_date` for eab be indented underneath the `eab` value?